src/libraries/oob-src.proj

```diff
@@ -32,7 +32,8 @@
 
     <!-- Don't build meta-projects in the NetCoreAppCurrent vertical. -->
     <ProjectReference Remove="Microsoft.Internal.Runtime.AspNetCore.Transport\src\Microsoft.Internal.Runtime.AspNetCore.Transport.proj;
-                              Microsoft.Internal.Runtime.WindowsDesktop.Transport\src\Microsoft.Internal.Runtime.WindowsDesktop.Transport.proj" />
+                              Microsoft.Internal.Runtime.WindowsDesktop.Transport\src\Microsoft.Internal.Runtime.WindowsDesktop.Transport.proj;
+                              Microsoft.Internal.Runtime.DotNetApiDocs.Transport\src\Microsoft.Internal.Runtime.DotNetApiDocs.Transport.proj" />
```

Comment:
Ok I see: we only build these in all configurations, but not in src.

Questions:

- The existing transport project for aspnetcore already includes ref assemblies and xmls?
- I don't see the windowsdesktop one here, but I assume it already existed. Does also include xmls?

Reply:
> I don't see the windowsdesktop one here, but I assume it already existed.

It's in the list above. It includes ref assemblies, lib assemblies and xml docs.

> The existing transport project for aspnetcore already includes ref assemblies and xmls?

Yes, the aspnetcore package includes ref assemblies, lib assemblies and xml docs.